Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1222,6 +1222,20 @@ working, take a look at `this KEP
     free mode, make sure that default Kubernetes services like ``kube-dns`` and ``kubernetes``
     have the required label value.
 
+Neighbor Discovery
+******************
+
+When kube-proxy replacement is enabled Cilium does L2 neighbor discovery of nodes in the cluster.
+In some rare cases Cilium may leave stale entries behind in the neighbor table causing packets
+between some nodes to be dropped. To prevent Cilium from performing the neighbor discovery and
+instead rely on the Linux kernel to discover hosts on the same L2 network you can pass the
+`--enable-l2-neigh-discovery` flag to the cilium-agent setting it to false. Relying on the Linux
```

Comment:
```suggestion
``--enable-l2-neigh-discovery`` flag to the cilium-agent setting it to false. Relying on the Linux
```
We use RST markup, and we need double-backquotes instead of one like with Markdown.

Reply:
On the same sentence, I'd maybe rephrase to have `false` closer to the option name itself. The risk is that users might read the sentence to fast and stop after `you can pass the --enable-l2-neigh-discovery flag to the cilium-agent`.

Reply:
@qmonnet can you propose a way through which I can rephrase the sentence?

Reply:
How about:
```
To prevent [...] you can set the ``--enable-l2-neigh-discovery`` flag to ``false`` in the agent's configuration.
```
or
```
To prevent [...] you can pass ``--enable-l2-neigh-discovery=false`` to the cilium-agent.
```
or
```
To prevent [...], the cilium-agent has a ``--enable-l2-neigh-discovery`` flag that you can set to ``false``.
```
or just swapping the mention to the agent and the value:
```
To prevent [...] you can pass the ``--enable-l2-neigh-discovery`` flag, setting it to ``false``, to the cilium-agent.
```
Whichever sounds best to you.

Reply:
addressed in https://github.com/cilium/cilium/commit/81553b9a88f7c2948666a721556895adb3bc6990.

Thanks!